openshift/fluentd.dc.yaml

```diff
@@ -0,0 +1,308 @@
+---
+apiVersion: v1
+kind: Template
+labels:
+  app: ${APP_NAME}-${INSTANCE}
+  env: ${LABEL_ENV}
+  role: ${APP_NAME}
+  template: ${APP_NAME}-${INSTANCE}-template
+metadata:
+  name: ${APP_NAME}-${INSTANCE}-template
+objects:
+  - apiVersion: v1
+    kind: ConfigMap
```

Comment:
Strongly suggest splitting the ConfigMap definition out of the deployment -> you can have multiple deployments of the same thing, but only have one defined configmap that services them all. There is no reason they should be defined together in the same template as they can potentially have different lifecycles. I suggest creating a new template called `fluentd.cm.yaml` to store just the template for configmap object and have it be invoked separately in readme documentation as part of CM/Secret setup.